samples/package.json

```diff
@@ -7,12 +7,14 @@
     "node": ">=10"
   },
   "files": [
-    "*.js"
+    "*.js",
+    "!test"
   ],
   "scripts": {
     "test": "c8 mocha --timeout 600000 test/*.js"
   },
   "dependencies": {
+    "@google-cloud/storage": "^5.4.0",
```

Comment:
I think this should go into "devDependencies"

Reply:
Done